cmd/monaco/integrationtest/rewrite_functions.go

```diff
@@ -35,42 +35,46 @@ func GetAddSuffixFunction(suffix string) func(line string) string {
 }
 
 func ReplaceName(line string, idChange func(string) string) string {
+	if strings.HasSuffix(line, "#monaco-test:no-replace") {
```

Comment:
So if there is a space after the comment, this won't work, but both approaches are fine for me.

Reply:
That's by design - it must be exactly this line ending,